For TSE review

javierbotero opened the pull request:
The project until now looks completed and working as expected. Kindly review and revert.

javierbotero committed:
Added the set up of the project
------------

All the files were added and the linters in the repo and locally.

javierbotero committed:
Corrected setup with the gitignore and the stylelint.config.js where put in the root

paulo-techie committed:
Add article text

javierbotero committed:
Finish adding text and put icons
------------
The text was narrow down and the icons were added and asjusted the shapes.

paulo-techie committed:
Add initial thumbnail pictures

javierbotero committed:
Added navbar and floated

----------

Adjustment to the navbar should be done. The layout need improvement.

paulo-techie committed:
Add nav bar and position it

javierbotero committed:
Add layout with

paulo-techie committed:
align main content

javierbotero committed:
Layout was set for the asides elements
--------------------------

The asides elements were floated to the right. The nav menu at the bottom needs improvements.

paulo-techie committed:
add bottom menu items

javierbotero committed:
Improvments of the layout

----------------

Improvements in the sidebar were done and spaces between the divs were added

paulo-techie committed:
add inflation-theory background image

javierbotero committed:
Add layout for the bottom nav bar
----------------------

The layout was acomplish by giving flex property and using flex direction of column in order to get the main divs organized. Some of the orther content use the same flex property with slightly different properties.

paulo-techie committed:
Add content and style to the inflation theory container

javierbotero committed:
Added Button and the information in the aside element at the top

paulo-techie committed:
finish styling the nav-bar

javierbotero committed:
Styles for font inside the figures are added

paulo-techie committed:
style author date icons and inflation theory header

javierbotero committed:
Correcting the layout og main and adjusting the style of the aside sign up

paulo-techie committed:
style sign-up button and editor's picks aside

javierbotero committed:
Added little adjustments to the side section

paulo-techie committed:
Finish entry of all text and html content

paulo-techie committed:
style inflation-theory tab

paulo-techie committed:
style bottom menu container and items

javierbotero committed:
Improvements in the more articles layout

javierbotero committed:
Merge branch 'article-text' of github.com:javierbotero/NewYorkTimes-Article into article-text

javierbotero committed:
Add styles for the bottom menu
-------------
Backgrounds colors were removed and the general style was impemented, just the sticky footer needs to be completed.

paulo-techie committed:
Make footer and correct typos

paulo-techie committed:
Fix linter errors using stylelint

paulo-techie committed:
Fix html validation and refix css linter errors

javierbotero committed:
Add README.md information and a picture for a screenshot of the webpage

javierbotero committed:
Merging linting errores corrections and README.md file

javierbotero committed:
File .stickler.yml is added

javierbotero committed:
Fixing issues with the linting process, the stylelint.config.js is added

javierbotero committed:
Added stylelint.config.js

javierbotero committed:
Merge pull request #2 from javierbotero/article-text

This for learning purposes --

izaiasneto4 commented on README.md:
```diff
@@ -1 +1,40 @@
+
+> One paragraph statement about the project.
+
+This project is the cloning content and appearance of the New York Times web page. The main idea was to use the float property and all its features. It also was allowed to use other CSS tools like the flex property.
```
Please, let's change this line from your README.md. In this project, we're not supposed to use the flex property. :wink: 

![image](https://user-images.githubusercontent.com/5157985/70670206-2cf1d480-1c57-11ea-944b-e716b34d50e0.png)

izaiasneto4 commented on css/style.css:
```diff
@@ -0,0 +1,462 @@
+}
+
+.footer-more {
+  display: flex;
```
Great work with your CSS! But let's keep in mind that for this project we're required to use **floats** and not the flex property! :+1: 

 
![image](https://user-images.githubusercontent.com/5157985/70671183-e487e600-1c59-11ea-8099-f87a4b526034.png)

- [x] Let's replace all of the flex property instances with float and also it's sub-properties in your style.css file :wink:

izaiasneto4 commented on index.html:
```diff
@@ -0,0 +1,656 @@
+          </div>
+        </nav>
+
+        <div class="main-content">
```
This one is optional but worth doing. Let's use the `<main>` HTML tag instead of the div with the class of main here. This will make your code [more readable, accessible and semantic](https://medium.com/@zac_heisey/7-alternatives-to-the-div-html-tag-7c888c7b5036)! :muscle:

izaiasneto4 reviewed (commented):
### Hi Folks! Great work so far with your project, you're almost done with it and your project looks awesome! :rocket: We just need to change some last things before being approved, check the comments below :arrow_down: 

Please, when finished with the requested changes, re-submit your project for a new review! :pray:

paulo-techie committed:
Replace flex with float, only bottom nav and menu remaining

paulo-techie committed:
Merge branch 'Feature' of https://github.com/javierbotero/NewYorkTimes-Article into article-text

javierbotero committed:
Changes are corrected with the README.md file and the property flex was removed
---------------
The flex property is removed and changed for the float property ion all the css file, also the linting process was succesfully done.

javierbotero committed:
Merge pull request #4 from javierbotero/clearfloat

Changes and corrections were done

imhta reviewed (approved):
# Project approved 🎉 

### Hello  @javierbotero @paulo-techie , 👋 You have done a great job in this project 👏 

<img src="https://thumbs.gfycat.com/SpeedyAmazingIndianpalmsquirrel-size_restricted.gif">


----
### Happy coding 😄 
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__

paulo-techie committed:
Fix bottom menu item list

Removed {.bottom-menu-items li} from css to fix the bottom menu

paulo-techie commented:
Project completed. Merged to master.
